tests/testthat/test-child-model.R

```diff
@@ -11,7 +11,8 @@ test_that("child model can be run for all years", {
       "p_hiv_pop_natural_deaths", "h_hiv_adult", "h_art_adult",
       "h_hiv_deaths_no_art", "p_infections", "h_hiv_deaths_art",
       "h_art_initiation", "p_hiv_deaths", "hc1_hiv_pop", "hc2_hiv_pop",
-      "hc1_art_pop", "hc2_art_pop"
+      "hc1_art_pop", "hc2_art_pop", "hc1_noart_aids_deaths", "hc2_noart_aids_deaths",
+      "hc1_art_aids_deaths", "hc2_art_aids_deaths", "hc_art_num"
```

Comment:
Is there anything else we can meaningfully test about this data? Even something like they're non-zero?

Reply:
added tests